nativelib/src/main/resources/gc/commix/Marker.c

```diff
@@ -0,0 +1,380 @@
+#include <stdio.h>
+#include <setjmp.h>
+#include "Marker.h"
+#include "Object.h"
+#include "Log.h"
+#include "State.h"
+#include "headers/ObjectHeader.h"
+#include "datastructures/GreyPacket.h"
+#include "GCThread.h"
+#include <sched.h>
+
+extern word_t *__modules;
+extern int __modules_size;
+extern word_t **__stack_bottom;
+
+#define LAST_FIELD_OFFSET -1
+
+// Marking is done using grey packets. A grey packet is a fixes size list that
```

Comment:
Are packets actual linked lists?

Reply:
No, the packets are lists not linked lists. It says that in the comment.